src/app/project/ontology/default-data/default-properties.ts

```diff
@@ -155,6 +168,7 @@ export class DefaultProperties {
                 {
                     icon: 'functions',
                     label: 'Decimal',
+                    description: 'Floating point number',
```

Comment:
I would simply say "Decimal value". I've only used the term "Floating point number" as a developer.